.github/mergify.yml

```diff
@@ -5,7 +5,7 @@ pull_request_rules:
       - check-success~=lint
       - check-success~=build
       - check-success~=test
-      - -dependabot-update-type = version-update:semver-minor
+      - -dependabot-update-type = version-update:semver-major
```

Comment:
Just checking, we use this so that we get both `patch` and `minor` updates right?

Reply:
Yeap, see original pr for details!